src/styles/_nibble.less

```
@@ -65,219 +31,49 @@
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: flex;
-  -webkit-box-align: center;
-  -webkit-align-items: center;
-  -ms-flex-align: center;
-  align-items: center;
+.nibble-shop-wrapper .list__item:nth-child(2n) {
+  margin-right: 0;
 }
 
-.nibble-shop-product-item-wrapper-title .title .icon {
-  margin-right: 15px;
-  width: 40px;
-  height: 40px;
-  display: -webkit-box;
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: flex;
-  -webkit-box-pack: center;
-  -webkit-justify-content: center;
-  -ms-flex-pack: center;
-  justify-content: center;
-  -webkit-box-align: center;
-  -webkit-align-items: center;
-  -ms-flex-align: center;
-  align-items: center;
-  border-radius: 50%;
+.nibble-shop-wrapper .list__item:nth-child(1),
+.nibble-shop-wrapper .list__item:nth-child(2) {
+  margin-top: 0;
 }
 
-.nibble-shop-product-item-wrapper-title .title .icon img {
-  border-radius: 50%;
-  -o-object-fit: cover;
-  object-fit: cover;
+.nibble-shop-wrapper .list__item a {
   display: block;
-  width: 100%;
-}
-
-.nibble-shop-product-item-wrapper-title .button {
-  margin-left: auto;
 }
 
-.nibble-shop-product-item-wrapper-title .button button {
-  width: 40px;
-  height: 40px;
-  border-radius: 50%;
-  display: -webkit-box;
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: flex;
-  -webkit-box-align: center;
-  -webkit-align-items: center;
-  -ms-flex-align: center;
-  align-items: center;
-  -webkit-box-pack: center;
-  -webkit-justify-content: center;
-  -ms-flex-pack: center;
-  justify-content: center;
-  background-color: #f1f1f1;
-}
-
-.nibble-shop-product-item-wrapper-banner {
-  margin-top: 40px;
-  overflow: hidden;
+.nibble-shop-wrapper .list__item a video {
+  display: block;
   width: 100%;
-  display: -webkit-box;
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: flex;
-  -webkit-box-pack: justify;
-  -webkit-justify-content: space-between;
-  -ms-flex-pack: justify;
-  justify-content: space-between;
-  -webkit-box-align: center;
-  -webkit-align-items: center;
-  -ms-flex-align: center;
-  align-items: center;
   -webkit-box-shadow: 0 4px 16px rgba(0, 0, 0, 0.12);
   box-shadow: 0 4px 16px rgba(0, 0, 0, 0.12);
   border-radius: 8px;
-  max-height: 280px;
-  background: #d8dfe6;
-}
-
-.nibble-shop-product-item-wrapper-banner video {
-  overflow: hidden;
-  display: block;
-  width: 100%;
-  margin: 0 auto;
-  max-width: 391px;
-}
-
-.nibble-shop-product-item-wrapper-info {
-  margin-top: 40px;
-}
-
-.nibble-shop-product-item-wrapper-info span {
-  font-family: 'Medium', sans-serif;
-  font-weight: 500;
-  font-size: 16px;
-  line-height: 24px;
-  color: rgba(60, 60, 67, 0.6);
+  height: 280px;
+  -o-object-fit: cover;
+  object-fit: cover;
 }
 
-.nibble-shop-product-item-wrapper-info h3 {
-  font-family: 'Bold', sans-serif;
-  font-weight: 700;
-  font-size: 24px;
-  line-height: 32px;
-  margin-top: 8px;
+.nibble-shop-wrapper .list__item a h3 {
+  font-family: 'Medium';
```

Please update here & for other similar items:
```suggestion
  font-family: 'Medium', sans-serif;
  font-weight: 500;
```

That's important in case of font absence (helps to facilitate the better UX approach)